server/routes/v1/posts.js

```diff
@@ -17,4 +17,12 @@ router.patch("/posts", middleware.apiAuth, exists, post.updatePost);
 
 router.delete("/posts", middleware.apiAuth, exists, post.deleteById);
 
+// post activity
+router.get("/posts/:post_id/activity", post.activity.get)
+
+// post comment
+router.post("/posts/:post_id/comments", middleware.apiAuth, post.comments.create);
+router.put("/posts/:post_id/comments/:comment_id", middleware.apiAuth, post.comments.update);
+router.delete("/posts/:post_id/comments/:comment_id", middleware.apiAuth, post.comments.destroy);
```

Comment:
Codacy has a fix for the issue: [Replace `"/posts/:post_id/comments/:comment_id",·middleware.apiAuth,·post.comments.destroy` with `⏎↹"/posts/:post_id/comments/:comment_id",⏎↹middleware.apiAuth,⏎↹post.comments.destroy⏎`](https://app.codacy.com/gh/logchimp/logchimp/pullRequest?prid=7771592)
```suggestion
router.delete(
```